lib/src/executables/intl_message_migration.dart

```diff
@@ -210,14 +212,32 @@ void main(List<String> args) async {
   }
 }
 
+void printDeprecationNotice() {
+  printInRed(
+      '***** Attention Users of over_react_codemod:intl_message_migration *****');
```

Comment:
I think just ***** Deprecation Notice ***** is enough.
And then in the next line, just say
"# over_react_codemod:intl_message_migration is deprecated and will ...

Reply:
Updated the messages